docs/user-guides/manual-setup.md

```diff
@@ -0,0 +1,162 @@
+The Genesis UTXO has to be spendable by the payment key.
+This command initializes governance of the chain as M of N multisig parametrized by GOVERNANCE_AUTHORITY and THRESHOLD parameters.
+
+Note: for more details please read [smart-contracts commands documentation](../../toolkit/smart-contracts/commands/readme.md).
+
+### Discover smart-contracts addresses and script hashes
+
+Use `<node-executable> smart-contracts get-scripts -c <GENESIS_UTXO>` to obtain Cardano addresses and scripts hashes.
+This guide assumes that the output is saved to `addresses.json` file.
+
+### Establish bootnodes
+
+The wizard prepares one bootnode address that is derived from the generated node network key and user input.
+This guide does not cover bootnodes as it is not a partner chains specific topic.
+Partner chains do not add any requirements nor capabilities in regards to bootnodes.
+
+## Create chain-spec file
+
+`<node-executable> wizards create-chain-spec` uses data from the previous step to generate chain-spec file.
+It is important to understand that most of chain-spec file generation is delegated to `<node-exectuable>` itself.
+Specifically, the wizard sets some specific environment variables and assumes that `<node-executable> build-spec` will use and that it will not require any other variables.
+Because what `<node-exectuable> build-spec` uses and requires is finally in control of the chain builder, the following command can only be seens as an example:
+```bash
+export COMMITTEE_CANDIDATE_ADDRESS=$(jq -r '.addresses.CommitteeCandidateValidator' addresses.json)
+export D_PARAMETER_POLICY_ID=$(jq -r '.policyIds.DParameter' addresses.json)
+export PERMISSIONED_CANDIDATES_POLICY_ID=$(jq -r '.policyIds.PermissionedCandidates' addresses.json)
+export ILLIQUID_SUPPLY_VALIDATOR_ADDRESS=$(jq -r '.addresses.IlliquidCirculationSupplyValidator' addresses.json)
+export NATIVE_TOKEN_POLICY_ID='0x00000000000000000000000000000000000000000000000000000000'
+export NATIVE_TOKEN_ASSET_NAME=''
+```
+Native token policy and asset name are out of partner chains control, they should be known by the chain builder.
+For chains that do not intent to use them, it is assumed that `<node-exectuable> build-spec` will not need them.
+
+Run `<node-executable> build-spec --disable-default-bootnode > chain-spec.json`.
+As stated above, the content of the file depends mostly on `<node-executable>`.
+
+Please consult our [intro doc chain-spec](../intro.md#chain-spec.json) section and update all the required fields, most notably:
+* `genesisUtxo` and `slotsPerEpoch` of Sidechain Pallet
+* `initialValidators` of Partner Chains Session Pallet
+* `initialAuthorities` of Session Validator Management Pallet
+* all occurrances `mainChainScripts`
+
+## Setup D-parameter and permissioned Permissioned Candidates
+
+`<node-executable> wizards setup-main-chain-state` guides user through creating (and conditionally submitting) two transactions that set up smart-contracts state.
```

Comment:
I think we should make a clear distinction on what this **guide** is about. We seem to be oscillating between describing what wizards do and outlining how to set up a chain without wizards.

I think we want the latter. So we would mostly just explain that wizards exist and can be used but then go on to to explain the necessary steps to create a chain as if wizards didn't exist at all - in a way that someone who hasn't done this before would be able to follow along and do the same. WDYT?